tests/travis/_config.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
lol, OK - that's reaaaaally old then. Might have actually been the first SilverStripe module ever to get travis support? Do you just want to copy the `.travis.yml` from another module and run with it? Much simpler: https://github.com/silverstripe/silverstripe-blog/blob/1.0/.travis.yml